src/_gettsim/transfers/kinderbonus/kinderbonus.py

```diff
@@ -1,12 +1,18 @@
-def kinderbonus_m(kindergeld_m: float, kindergeld_params: dict) -> float:
+"""Kinderbonus."""
+
+from _gettsim.functions.policy_function import policy_function
+
+
+@policy_function(start_date="2020-01-01", end_date="2021-12-31")
+def betrag_m(kindergeld__betrag_m: float, kindergeld_params: dict) -> float:
```

Comment:
```suggestion
def betrag_y(kindergeld__betrag_m: float, kindergeld_params: dict) -> float:
```
Or is there any reason why we would do that? I'd think it is simply a leftover of pre-automatic-conversion days.